client/src/components/Form/Form.js

```diff
@@ -1,43 +1,74 @@
-import React, { useState, useEffect } from "react";
+import React, { useState, useEffect } from 'react';
+import clsx from 'clsx';
 
 // Material-UI Imports
-import { createStyles, makeStyles } from "@material-ui/core/styles";
-import Button from "@material-ui/core/Button";
-import InputLabel from "@material-ui/core/InputLabel";
-import MenuItem from "@material-ui/core/MenuItem";
-import FormControl from "@material-ui/core/FormControl";
-import Select from "@material-ui/core/Select";
+import { createStyles, makeStyles } from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
 
 // Local imports
-import DistanceSlider from "./DistanceSlider";
-import CheckboxGroup from "./CheckboxGroup";
+import DistanceSlider from './DistanceSlider';
+import CheckboxGroup from './CheckboxGroup';
+
+import data from '../../data/demo.json';
+
+function haversineCalc(la1,la2,lo1,lo2){

```

Comment:
We already have geolocation baked into the map, lets either tap into that or replace it so we don't have 2 implementations that could get out of sync and harder to maintain.

Reply:
i can tap into the map geolocation perhaps for the current users location but the haversineCalc is there to calculate the filtered list to render to the map

Reply:
Yep makes sense, I just don't want to have 2 implementations of geolocation. Lets break the Haversine out into a separate file then, separation of concerns and what not

Reply:
Maybe move it to utils?

Reply:
Or just into a sep file in the Form dir?

Reply:
I'd say utils makes more sense.